src/autoscaler/eventgenerator/generator/evaluator.go

```diff
@@ -70,6 +70,10 @@ func (e *Evaluator) doEvaluate(triggerArray []*model.Trigger) {
 			return
 		}
 		for _, appMetric := range appMetricList {
+			if appMetric.Value < 0 {
+				e.logger.Info("should not send trigger alarm to scaling engine because there is nil-value metric", lager.Data{"trigger": trigger, "appMetric": appMetric})
```

Comment:
who is this for?
if our appMetric.Value is less than 0, why would an admin care? What are they supposed to do when they see this?